pkg/config/engine/api.go

```diff
@@ -19,7 +19,7 @@ package engine
 // Interface defines the API for a runtime config updater.
 type Interface interface {
 	DefaultRuntime() string
-	AddRuntime(string, string, bool, ...map[string]interface{}) error
+	AddRuntime(string, string, bool) error
```

Comment:
Question: why remove this?

Reply:
This was added to allow users to override the `SystemdCgroup` option but we decided not to actually follow this as a solution. Including this in the API overcomplicates the implementation and I would rather remove it as unused instead of keeping the additional complexity.

Reply:
Understood